src/commands/dev/dev.js

```diff
@@ -334,6 +354,7 @@ const createDevCommand = (program) => {
         'specify the path to a local GeoIP location database in MMDB format',
       ).hideHelp(),
     )
+    .addOption(new Option('--graph', 'enable Netlify Graph support').hideHelp())
```

Comment:
is this only during the internal beta that it is hidden?

Reply:
Well, until it's ready for a wider/announcement release in general